src/factories/balance.factory.ts

```diff
@@ -1,12 +1,17 @@
-import * as Factory from "factory.ts";
-
 import { Balance } from "..";
 import { tokenFactory } from "./token.factory";
 
-export const balanceFactory = Factory.Sync.makeFactory<Balance>({
+export const defaultBalance: Balance = {
```

Comment:
I like this! It serves the same purpose as factory.ts but we don't need to have the dependency :)

Reply:
Awesome, so I've refactor all mocked objects with https://github.com/yearn/yearn-sdk/pull/218/commits/62a023c2795ed43d426a8db55c6a249ed74d4691

Removed factory.ts as a dependency with https://github.com/yearn/yearn-sdk/pull/218/commits/df30f983229d159edef2aaa505094d47e5d5a388

Thanks for the review @FiboApe 🙇